client/src/components/Draftail/blocks/EmbedBlock.js

```diff
@@ -4,16 +4,20 @@ import React from 'react';
 import { STRINGS } from '../../../config/wagtailConfig';
 
 import MediaBlock from '../blocks/MediaBlock';
+import { shortenLabel } from '../decorators/TooltipEntity'
```

Comment:
```suggestion
import { shortenLabel } from '../decorators/TooltipEntity';
```

Missing a semicolon here.